src/components/triggerField/TriggerField.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import SearchField from "../searchField/SearchField";
+
+import Filter from "@material-ui/icons/FilterList";
+import IconButton from "@material-ui/core/IconButton/index";
+import InputAdornment from "@material-ui/core/InputAdornment/index";
+import Popover from "@material-ui/core/Popover/index";
+
+/**
+ * @author aramsey
+ *
+ * A search field that has a customizable "trigger". When the trigger is clicked,
+ * the component's children will be displayed in a popover.
+ */
+class TriggerField extends Component {
```

Comment:
Do we still want to keep this component?  I don't believe we're using it anywhere.